### ISSUE TITLE
Bug Report: line scroll speed doesn’t change when the bpm is changed (charting)

### ISSUE THREAD

**Issue by Armodillomatt12** [KadeArchive/Kade-Engine#1530]
when i went to redo the charting, the notes that were inside the boxes now appear outside more than it should be.
pico’s charting have their notes in the boxes but for a song like milf, the boxes don’t line up with the bpm.
it seems that the line scroll when playing through the song doesn’t change according to the song’s bpm. it syncs up well with pico because of it being 150 bpm. anything other than that bpm ratio would not sync up with the boxes.

tl;dr
the scroll speed when charting doesn’t match the bpm it is set to.

**Comment by KlutchDj** [comment]
that's now how the bpm change works. its not suppose to change the scroll speed because some of us may have a universal scroll speed. heres more explanation of how BPM change works. https://github.com/KadeDev/Kade-Engine/issues/1440#issuecomment-886002360

**Comment by Armodillomatt12** [comment]
> that's now how the bpm change works. its not suppose to change the scroll speed because some of us may have a universal scroll speed. heres more explanation of how BPM change works. [#1440 (comment)](https://github.com/KadeDev/Kade-Engine/issues/1440#issuecomment-886002360)

i'm not sure if you're referring to the charting line scroll speed or the in game scroll speed

**Comment by Armodillomatt12** [comment]
a screenshot of roses's charting
expected: the arrows to be aligned with the boxes and the playback speed to change depending on the bpm of the song like it did prior
instead. the arrows are's placed properly according to the boxes and the line speed is set at a specific speed
![image](https://user-images.githubusercontent.com/25271478/127726395-f6b2577b-d760-47cf-a2ab-3aab1def4437.png)

**Comment by isakube** [comment]
> a screenshot of roses's charting
> expected: the arrows to be aligned with the boxes and the playback speed to change depending on the bpm of the song like it did prior
> instead. the arrows are's placed properly according to the boxes and the line speed is set at a specific speed
> ![image](https://user-images.githubusercontent.com/25271478/127726395-f6b2577b-d760-47cf-a2ab-3aab1def4437.png)

the grid is aligned to 150 bpm. if you look in Spookeez you can see that.

**Comment by Kade-github** [comment]
the grid isn't technically aligned at all, and really I was going to remove it all together. in 1.6.1 an option will be there to enable it/disable it.
